styles/index.css

```diff
@@ -0,0 +1,109 @@
+.main-layout {
+  padding: 4px;
+}
+
+.main-layout > * {
+  padding: 2px;
+
+  border: 2px dashed lightblue;
+}
+
+.center-wrapper {
+  max-width: 960px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin: 0 auto;
+}
+
+.two-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.three-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+}
+
+.three-row-grid__expand-two {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.gap-sm {
+  gap: 4px;
+}
+
+.main-header {
+  width: 275px;
+  height: 100vh;
+}
+
+.main-header > * {
+  border: 2px dashed lightcoral;
+}
+
+.resources-sidebar {
+  width: 275px;
+}
+
+/* MAIN LAYOUTS */
+
+.button {
+  display: block;
+}
+
+.nav-list {
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+}
+
+.nav-list--link {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  gap: 8px;
+}
+
+@media screen and (max-width: 800px) {
+  .center-wrapper {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  .main-header {
+    width: auto;
+  }
+
+  .nav-list--link {
+    grid-template-columns: auto;
+
+    gap: 0;
+  }
```

Comment:
@Tibor22 love what you said about"re-usable classes sometimes aren't fit for purpose".

This was a good example of having to create a custom class to get the desired effect.